staff/templates/form_overview.html

```diff
@@ -2,46 +2,49 @@
-                <th></th>
-              </tr>
-              {% for distribution in distributions %}
-                <tr>
-                  <td>{{ form_name }}</td>
-                  <td>{{ distribution.user.first_name }} {{ distribution.user.last_name }}</td>
-                  <td>{{ distribution.submitted }}</td>
-                  <td>
-                    <form method="POST">
-                      <div>
-                        {% csrf_token %}
-                        <small>Submitted: &ensp; {{ form.submitted }} &emsp;&emsp;</small>
-                        <button class="mui-btn mui-btn mui-btn--primary" type="submit">Submit</button>
-                        <input type="hidden" name="form_name" value="{{ form_name }}" />
-                        <input type="hidden" name="user_id" value="{{ distribution.user.id }}" />
-                      </div>
-                      </form>
-                    </td>
+            <div style="text-align: center">
+              <div style="display: inline-block; width: 85%" class="mui-panel">
+                <table class="mui-table mui-table--bordered">
+                  <h3 style="padding-left: 1%">{{ form_name }}</h3>
+                  <tr>
+                    <th style="text-align: center">Form Name</th>
+                    <th style="text-align: center">Distribution</th>
+                    <th style="text-align: center">Submitted</th>
+                    <th></th>
                   </tr>
-                {% endfor %}
-              </table>
-              <form action={% url 'notify_unsubmitted_users' %} method="GET">
-                <input type="hidden" name="notify_unsubmitted_users" value="{{ form_name }}" />
-                <br><td><button class="mui-btn mui-btn mui-btn--primary" type="submit">Notify Unsubmitted Users</button></td>
-              </form>
+                  {% for distribution in distributions %}
+                    <tr>
+                      <td>{{ form_name }}</td>
+                      <td>{{ distribution.user.first_name }} {{ distribution.user.last_name }}</td>
+                      <td>{{ distribution.submitted }}</td>
+                      <td>
+                        <form method="POST">
+                          <div>
+                            {% csrf_token %}
+                            <small>Submitted: &ensp; {{ form.submitted }} &emsp;&emsp;</small>
+                            <button class="mui-btn mui-btn mui-btn--primary" type="submit">Submit</button>
+                            <input type="hidden" name="form_name" value="{{ form_name }}" />
+                            <input type="hidden" name="user_id" value="{{ distribution.user.id }}" />
+                          </div>
+                          </form>
+                        </td>
+                      </tr>
+                    {% endfor %}
+                  </table>
+                  <form action={% url 'notify_unsubmitted_users' %} method="GET">
```

Comment:
This can be a link

Reply:
Resolved 6eb5cb979b3e7f5677a52dfe3d696b44155aa78e